kod/object/passive/spell.kod

```diff
@@ -795,29 +795,24 @@ messages:
       }
 
       % Moved from CanPayManaVigor, because it doesn't relate to paying vigor 
-      %  or mana!  Spells cast from items should follow Karma restrictions
-
-      % If the spell is being cast by a made item then there are no Karma
-      %  restrictions.
-      if NOT (bMade OR Send(self,@KarmaCheck,#who=who))
+      % or mana!  Spells cast from items should follow Karma restrictions
```

Comment:
This seems like a pretty big change!  Are people OK with it?

Reply:
It's not as big as you might think, given the way things behave on live.  Currently on live, spellitems have the following behaviors in relation to karma:

- All spellitems not created by distill care about karma, whether they are flagged to or not.  This is a major reason why remove curse, cure poison, and cure disease potions bought from NPCs or dropped by monsters fail when the expectation is that they shouldn't.
- Rescue potions are the only spellitem actually flagged to take karma into consideration for some reason.  (A line of text is appended to the description of items flagged with a karma requirement, so this behavior is transparent to players.)  Personally, I think more of these items should be flagged for karma restrictions and would be happy to adjust that in a separate PR.

So in short, all this change really does is allow spellitems to start working correctly, as dropped or bought potions will behave as expected.  The check for made (distilled) items honestly didn't make a lot of sense anyway.